core/mondoo-aws-security.mql.yaml

```diff
@@ -2264,10 +2264,7 @@ queries:
     impact: 50
     filters: |
       asset.platform == "aws-sagemaker-notebookinstance"
-    mql: |
-      aws.sagemaker.notebookInstance
-        .where(asset.name == name)
-        .all(details.kmsKey != empty)
+    mql: aws.sagemaker.notebookinstance.details.kmsKey != empty
```

Comment:
```suggestion
    mql: |
      aws.sagemaker.notebookInstances
        .where(asset.name == name)
        .all(details.kmsKey != empty)
```

Reply:
Think the sagemaker resource is still broken, BUT the work-around check was also wrong:

![image](https://github.com/mondoohq/cnspec-policies/assets/112621871/4aed5fb8-bb63-43fc-9964-8aafbaa761b7)